src/compile/selection/point.ts

```diff
@@ -1,11 +1,49 @@
-import {Stream} from 'vega';
+import {Signal, Stream} from 'vega';
 import {stringValue} from 'vega-util';
-import {SelectionCompiler, TUPLE, unitName} from '.';
+import {SelectionCompiler, TUPLE, isTimerSelection, unitName} from '.';
 import {SELECTION_ID} from '../../selection';
 import {vals} from '../../util';
 import {BRUSH} from './interval';
 import {TUPLE_FIELDS} from './project';
 
+export const CURR = '_curr';
+
+const animationSignals = (selectionName: string, scaleName: string): Signal[] => {
+  return [
+    // timer signals
+    {
+      name: 'anim_clock',
+      init: '0',
+      on: [
+        {
+          events: {type: 'timer', throttle: 16.666666666666668},
```

Comment:
Where does this throttle come from? (I'm guessing it's something like the browser's requestNextAnimFrame?) I wonder if there's a way to not hardcode this value...

Reply:
this is 60 frames / second converted into milliseconds / frame. we could make that more legible by writing it as `1 / 60 * 1000` and adding a comment